src/Getintouch/getintouch.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styles from "./getintouch.module.css";
+
+export default function GetInTouch() {
+  return (
+    <div className={styles.container}>
+        <div className={styles.textContainer}>
+          <div className={styles.heading}>
+            <p>Get <span style={{color:"#F1A140"}}>In Touch</span></p>
+          </div>
+          <div className={styles.subheading}>
+            <p>Send us a message with your question and we’ll reply within 24 hours!</p>
+          </div>
+        </div>
+        <div className={styles.formContainer}>
+          <form>
+            <label>Name</label><br></br>
+            <input type="text" name="fname" placeholder="Vuyanzi Glorious"></input><br></br>
```

Comment:
@shades-7 Please use a generic name like `John Doe` and a generic email `johndoe@gmail.com`.

Reply:
Okay will update soon